<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>770</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Geometric Transformation</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_ImageList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image list</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>320</width>
       <height>340</height>
      </rect>
     </property>
     <property name="title">
      <string>Original Image</string>
     </property>
     <widget class="QLabel" name="label_OriImg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>370</y>
       <width>320</width>
       <height>340</height>
      </rect>
     </property>
     <property name="title">
      <string>Processed Image</string>
     </property>
     <widget class="QLabel" name="label_proImg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>201</width>
       <height>471</height>
      </rect>
     </property>
     <property name="title">
      <string>Function</string>
     </property>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>180</width>
        <height>430</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Trapezoidal</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_tra">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>161</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QSlider" name="verticalSlider_lx">
               <property name="maximum">
                <number>5</number>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="verticalSlider_rx">
               <property name="maximum">
                <number>5</number>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="verticalSlider_h">
               <property name="maximum">
                <number>5</number>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLCDNumber" name="lcdNumber_lx">
               <property name="digitCount">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber_rx">
               <property name="digitCount">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber_h">
               <property name="digitCount">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>L_x</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>R_x</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>h</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Wavy</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>161</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QSlider" name="verticalSlider_cir_t">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="verticalSlider_cir_h">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLCDNumber" name="lcdNumber_cir_t">
               <property name="digitCount">
                <number>1</number>
               </property>
               <property name="value" stdset="0">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber_cir_h">
               <property name="digitCount">
                <number>1</number>
               </property>
               <property name="value" stdset="0">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>T</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>H</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_wavy">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Circular</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_cir">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Image Fusion</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>250</width>
       <height>270</height>
      </rect>
     </property>
     <property name="title">
      <string>First Image</string>
     </property>
     <widget class="QLabel" name="label_choose_f">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>230</width>
        <height>230</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_chooseF">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>choose</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>250</width>
       <height>270</height>
      </rect>
     </property>
     <property name="title">
      <string>Second Image</string>
     </property>
     <widget class="QLabel" name="label_choose_s">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>230</width>
        <height>230</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_chooseS">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>choose</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>541</width>
       <height>411</height>
      </rect>
     </property>
     <property name="title">
      <string>Fusion Image</string>
     </property>
     <widget class="QLabel" name="label_fusionimg">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>370</width>
        <height>370</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_imgFusion">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>287</y>
       <width>541</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Image fusion</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Hough Transform</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>320</width>
       <height>340</height>
      </rect>
     </property>
     <property name="title">
      <string>Original Image</string>
     </property>
     <widget class="QLabel" name="label_houghT_oriImg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_houghT_choose">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>choose</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>320</width>
       <height>340</height>
      </rect>
     </property>
     <property name="title">
      <string>Processed Img</string>
     </property>
     <widget class="QLabel" name="label_houghT_proImg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_houghT_APPLY">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
